src/utils/particle/updater.ts

```diff
@@ -110,14 +114,27 @@ export class Updater {
         const container = this.container;
         const options = container.options;
         const particle = this.particle;
-
-        if (container.interactivity.mouse.pos_x && options.interactivity.events.onhover.parallax.enable) {
+        const parallaxForce = options.interactivity.events.onhover.parallax.force;
+        const mousePos = {
+            x: container.interactivity.mouse.pos_x || 0,
+            y: container.interactivity.mouse.pos_y || 0,
+        };
+        const windowDimension = {
+            width: window.innerWidth / 2,
+            height: window.innerHeight / 2,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'height' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)